packages/material/src/controls/MaterialInputControl.tsx

```diff
@@ -61,23 +63,27 @@ export class MaterialInputControl extends Control<ControlProps, ControlState> {
       style.display = 'none';
     }
     const showDescription = !isDescriptionHidden(visible, description, this.state.isFocused);
-
-    return (
-      <FormControl
-        style={style}
-        fullWidth={!trim}
-        onFocus={this.onFocus}
-        onBlur={this.onBlur}
-      >
-        <InputLabel htmlFor={id} error={!isValid}>
-          {computeLabel(isPlainLabel(label) ? label : label.default, required)}
-        </InputLabel>
-        <DispatchField uischema={uischema} schema={schema} path={parentPath} />
-        <FormHelperText error={!isValid}>
-          {!isValid ? formatErrorMessage(errors) : showDescription ? description : null}
-        </FormHelperText>
-      </FormControl>
-    );
+    const field = _.maxBy(fields, r => r.tester(uischema, schema));
+    if (field === undefined || field.tester(uischema, schema) === -1) {
```

Comment:
A couple of things: 
1. Use `NOT_APPLICABLE` constant.
2. Are there any other controls that need to be updated to make the behavior consistent, e.g. `MaterialBooleanControl`? What about the vanilla renderer set? Ideally, this check could be moved into a [HOC](https://reactjs.org/docs/higher-order-components.html)
3. Add test case(s)